ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -249,6 +250,27 @@ def yield_pipeline_status(
                 " Skipping status ingestion."
             )
 
+    def get_schedule_interval(self, pipeline_data) -> Optional[str]:
+        """
+        Fetch Schedule Intervals from Airflow Dags
+        """
+        schedule_interval_val = pipeline_data.get("timetable", {}).get("__var", {})
+        if schedule_interval_val:
+            # Fetch Cron as String
+            return schedule_interval_val.get("expression", None)
+        # If the Schedule interval is a const value like @once, @yearly etc
+        # __type sends the module path, and once instantiated
+        try:
+            return import_from_module(
+                pipeline_data.get("timetable", {}).get("__type", {})
```

Comment:
did we try this against different airflow versions? If we grab super deep params there things might change 🤔